staging/src/k8s.io/apiserver/pkg/endpoints/handlers/patch.go

```diff
@@ -108,6 +108,12 @@ func PatchResource(r rest.Patcher, scope RequestScope, admit admission.Interface
 		userInfo, _ := request.UserFrom(ctx)
 		staticAdmissionAttributes := admission.NewAttributesRecord(nil, nil, scope.Kind, namespace, name, scope.Resource, scope.Subresource, admission.Update, userInfo)
 		admissionCheck := func(updatedObject runtime.Object, currentObject runtime.Object) error {
+			isNotZeroObject, err := hasUID(currentObject)
+			if err != nil {
+				return fmt.Errorf("unexpected error when extracting UID from oldObj: %v", err.Error())
+			} else if !isNotZeroObject {
```

Comment:
misunderstanding I think. Even remove the upper logic. Just a comment: // if we allow create-on-patch, TODO: ...

Reply:
@sttts Oops.. Indeed